portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/blogs/MyBlogTest.testcase

```diff
@@ -0,0 +1,80 @@
+@component-name = "portal-lima"
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given the Blogs Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "Blogs Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "Blogs Page",
+				widgetName = "Blogs");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+		}
+	}
+
+	@description = "Can Search Blogs"
+	@priority = 5
+	test CanSearchBlogs {
+		task ("And two Blog Entries exists") {
+			JSONBlog.addEntry(
+				entryContent = "Blog Entry Content 1",
+				entryTitle = "Blog Entry Title 1");
+
+			JSONBlog.addEntry(
+				entryContent = "Blog Entry Content 2",
+				entryTitle = "Blog Entry Title 2");
+
+			Navigator.gotoPage(pageName = "Blogs Page");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blog Entry Content 1",
+				entryTitle = "Blog Entry Title 1");
+
+			BlogsEntry.viewEntryPG(
+				entryContent = "Blog Entry Content 2",
+				entryTitle = "Blog Entry Title 2");
+		}
+
+		task ("When the User searches for Blog Entry one") {
+			SearchPortlets.searchEmbedded(searchTerm = "\"Blog Entry Title 1\"");
+		}
+
+		task ("Then Blog Entry two should display and Blog Entry 2 should not") {
+			SearchResultPortlet.viewSearchResults(
+				searchAssetTitle = "Blog Entry Title 1",
+				searchAssetType = "Blogs Entry",
+				searchTerm = "\"Blog Entry Title 1\"");
+
+			SearchResultPortlet.viewSearchResultNotPresent(
+				searchAssetTitle = "Blog Entry Title 2",
+				searchAssetType = "Blogs Entry");
+		}
+
+		task ("And when the User searches for Blog Entry two") {
+			SearchPortlets.searchEmbedded(searchTerm = "\"Blog Entry Title 2\"");
+		}
+
+		task ("Then Blog Entry two should display and Blog Entry 1 should not") {
```

Comment:
Use a name more similar as it's written, if you are using numbers, keep the usage to establish a standard: "Blog Entry 2"

Should be : Then Blog Entry 2 should display and Blog Entry 1 shouldn't